doc/nrf/protocols/matter/end_product/factory_provisioning.rst

```diff
@@ -13,6 +13,935 @@ The factory data parameter set includes different types of information, for exam
+|                        |                                      |                      |              |             | It is provided as a CBOR map type from persistent storage and should be parsed in the user application.                                                                                                                     |
+|                        |                                      |                      |              |             | This data is not used by the Matter stack.                                                                                                                                                                                  |
+|                        |                                      |                      |              |             | To learn how to work with user data, see the :ref:`ug_matter_device_factory_provisioning_how_to_set_user_data` section.                                                                                                     |
++------------------------+--------------------------------------+----------------------+--------------+-------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Factory data format
+===================
+
+The factory data set must be saved into a HEX file that can be written to the flash memory of the Matter device.
+
+In the nRF Connect example, the factory data set is represented in the CBOR format and is stored in a HEX file.
+The file is then programmed to a device.
+The JSON format is used as an intermediate, human-readable representation of the data.
+The format is regulated by the `Factory data schema`_ file.
+
+All parameters of the factory data set are either mandatory or optional:
+
+* Mandatory parameters must always be provided, as they are required for example to perform commissioning to the Matter network.
+* Optional parameters can be used for development and testing purposes.
+  For example, the ``user`` data parameter consists of all data that is needed by a specific manufacturer and that is not included in the mandatory parameters.
+
+In the factory data set, the following formats are used:
+
+* uint8, uint16, and uint32 - These are the numeric formats representing, respectively, one-byte length unsigned integer, two-bytes length unsigned integer, and four-bytes length unsigned integer.
+  This value is stored in a HEX file in the big-endian order.
+* Byte string - This parameter represents the sequence of integers between ``0`` and ``255``(inclusive), without any encoding.
+  Because the JSON format does not allow the use of byte strings, the ``hex:`` prefix is added to a parameter, and its representation is converted to a HEX string.
```

Comment:
```suggestion
* Byte string - This parameter represents the sequence of integers between ``0`` and ``255`` (inclusive), without any encoding.
  Because the JSON format does not allow the use of byte strings, the ``hex:`` prefix is added to a parameter, and its representation is converted to a HEX string.
```